lib/1.4/dml-builtins.dml

```diff
@@ -1171,7 +1171,10 @@ bank. The `aux` parameter is NULL by default, and is passed on to bank methods.
 In order to pass additional information on the access down to register and
 field methods, one can override `transaction_access`, decode needed information
 from the incoming transaction, and call `default` with the extracted
-information in the `aux` argument.
+information in the `aux` argument. Accesses that are bigger than 8 bytes are
+split on 8-byte address boundaries. For example, a 12-byte access to address 5
+will be split into one 3-byte access at address 5, one 8-byte access at address
+8, and one 1-byte access at address 16.
```

Comment:
Maybe we want to change this later, by extending the register dispatcher to handle accesses of arbitrary length. This will not happen in the near future (your fix solves all known problems in practice), but can we reformulate the documentation to disallow users from relying on getting accesses split in the middle of registers?

Also, it can be worth mentioning something about the relation to the read/write methods: transaction_access splits accesses before passing to `read` and `write`, which may split the access further along register boundaries.

Reply:
Hm, something like "accesses will be split in chunks of at most 8 bytes size"?